econml/dml.py

```diff
@@ -50,29 +78,32 @@ class _RLearner(LinearCateEstimator):
         If RandomState instance, random_state is the random number generator;
```

Comment:
Are we sure that whatever change in the fit logic we implemented in the fit methods was also propagated to the "score" method??? Weirdly the score method reimplements the fit logic separately and does not use the fit methods. I think this needs to change and the logic needs to be implemented only once to avoid having to keep track of changes at two places.

Reply:
I can refactor `score` slightly, but a lot of the logic is different enough from `fit` that it's hard to reuse most of the implementation even though there are conceptual similarities.  Is there something in particular that you're worried about?